randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -563,6 +581,10 @@ Extra - map_name: rm_a5c14
           Any of the following:
               Mid-Air Morph (Beginner) or Power Grip Wall
               Speed Booster and Shinesparking Tricks (Beginner)
+  > Pipe to The Tower
```

Comment:
I dont like duplicated info between door -> a4 pipe and a3 pipe -> a4 pipe

Reply:
oh yep, meant to remove that.